transcribed/page0270.txt

```diff
@@ -1,14 +1,14 @@
 001                                                                             PAGE 27
-002     058 005 DEVTAB: 200000+LPTO,,(SIXBIT /LPT/)     ;4.8 OUTPUT 4.9 INPUT
+002     058 005 DEVTAB: 200000+LPTO,,(SIXBIT /LPT/)    ;4.8 OUTPUT 4.9 INPUT
 003     060 004         600000+TTYO1,,(SIXBIT /TTY/)
 004     052 003         600000+NULO,,(SIXBIT /NUL/)
 005     053 004         600000+USRO,,(SIXBIT /USR/)
 006     002 033         400000+VIDI,,(SIXBIT /VID/)
 007
 008     027 002 NDEVS==.-DEVTAB
 009
-010     060 010 DVT1:   TTYO,,<SIXBIT /TY/>_-24.
-011     084 026         UTO,,<SIXBIT /UT/>_-24.
+010     060 010 DVT1:   TTYO,,<SIXBIT /TY/>--24.
+011     084 026         UTO,,<SIXBIT /UT/>--24.
 012
```

Comment:
Hard to see, but those are left arrows.  The syntax uses <> for arithmetic grouping and the arrow is the left shit operator.  A negative shift is a right shift.  Left arrow is spelled _ in ASCII 67 which we use here.